executor/executor.c

```diff
@@ -481,5 +481,6 @@ int main(int argc, char* argv[]) {
                 }
             }
         }
+		usleep(4000); //throttle this thread to ~200 loops/sec

```

Comment:
Make it 100,000 i.e. 10 Hz. The main loop doesn't need to be that responsive

Reply:
👌